### PR TITLE
Avatar component created

### PR DESCRIPTION
**Pull request by swstica**
Avatar component created and corrected some naming conventions

### CODE REVIEW

**Comment by varun1233** [comment]
A suggestion, try creating different css files for different components, you have added css for avatars and sidebar in one file only

**Review by rahulk31 — COMMENTED** [review]
Code looks good to me can be merged.  You can plan to have separate css files for each component and import them in your main css file. Could be easier to debug and review.

File: avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+.avatar-sm{
+    width: 6rem;
+    height: 6rem;
+}
+.avatar-xs{
+    width: 3rem;
+    height: 3rem;
+}
+.avatar-text{
```
Comment (veereshbv04) [inline avatar/avatar.css:42]: The size of text is same for all avatars, you should have created variations in different avatars

File: avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+.sidebar-heading{
+    padding: 1rem;
+}
+.component-items{
+    padding: 1rem;
+}
+.component-items:hover{
+    background-color: var(--secondary-color);
+    color: var(--primary-color);
+    border-radius: 0.5rem;
```
Comment (veereshbv04) [inline avatar/avatar.css:68]: for any units less than 1 it is recomended to use px, instead of rem

**Pull request merged**